src/components/application_manager/include/application_manager/commands/mobile/set_app_icon_request.h

```diff
@@ -87,10 +87,11 @@ class SetAppIconRequest : public CommandRequestImpl {
    * @brief Remove oldest icons
    * @param storage Path to icons storage
    * @param icons_amount Amount of icons to be deleted
+   * @param is_correct_removed_out Checks, if removing of icons was correct
    */
   void RemoveOldestIcons(const std::string& storage,
                          const uint32_t icons_amount,
-                         bool& is_correct_removed) const;
+                         bool& is_correct_removed_out) const;
```

Comment:
@VVeremjova  Also about "out_" prefix, please take a look at this article:  https://github.com/smartdevicelink/sdl_core/wiki/SDL-Coding-Style-Guide